jira-spec-artifacts.yaml

```diff
@@ -0,0 +1,21 @@
+# Content for https://github.com/HL7/JIRA-Spec-Artifacts/blob/master/xml/FHIR-uv-shc-vaccination.xml
+# Run script/artifacts-for-jira-spec to generate XML to STDOUT
+specification:
+  gitUrl: "https://github.com/HL7/fhir-shc-vaccinationm-ig"
+  url: "http://hl7.org/fhir/fhir-shc-vaccinationm-ig"
```

Comment:
typo

Reply:
do we want -ig in "url"?

Reply:
> do we want -ig in "url"?

https://github.com/HL7/fhir-mCODE-ig has `-ig` so I think that's the convention?